.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+  "editor.formatOnSave": false
+}
```

Comment:
this should probably be git-ignored

Reply:
maybe better in a different pr?

Reply:
Given that many people use vscode (many of the devs of this repo included), I'd be fine adding it 👍 no other PR needed.